apps/subscriptions/src/pages/SubscriptionsList.tsx

```diff
@@ -45,11 +43,35 @@ export const SubscriptionsList: FC = () => {
 
   return (
     <HomePageLayout title='Subscriptions'>
+      <SearchWithNav
+        queryString={queryString}
+        onUpdate={(qs) => {
+          navigate(`?${qs}`, {
+            replace: true
+          })
+        }}
+        onFilterClick={(queryString) => {
+          setLocation(appRoutes.filters.makePath({}, queryString))
+        }}
+        hideFiltersNav={false}
+      />
```

Comment:
Can we apply same patter you applied recently to hide search bar if list is empty and no filters have been applied?

Reply:
Good idea :)

Reply:
Please ignore my suggestion.
We realized this needs some changes in the `useResourceFilters` components and will be implemented in the future for all apps :)

Reply:
As reported by Giuse this change will be done as soon as it will be possibile to manage required infos in `useFilteredList` hook.